### PR TITLE
Minor proposal (line 99)

### PR DESCRIPTION
**Pull request by kant**
### Description

Minor proposal (or small fix) on documentation in line 99

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_pull_request_checklist.html)

### CODE REVIEW

**Comment by bjlittle** [comment]
@kant Thanks for taking the time to submit this PR, much appreciated.

Just need you to sign the CLA, and we're good to go 👍

File: docs/src/installing.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ local copy of Iris::
   conda activate iris-dev
 
 The ``--force`` option is used when creating the environment, this is optional
-and will force the any existing ``iris-dev`` conda environment to be deleted
+and will force that any existing ``iris-dev`` conda environment to be deleted
```
Comment (pp-mo) [inline docs/src/installing.rst:99]:
```suggestion
and will force any existing ``iris-dev`` conda environment to be deleted
```
It's good to change this, but I think the changed version reads wrong. 
I'd propose to just remove the exatr word entirely.
Do you think that works?

**Review by pp-mo — COMMENTED** [review]
Good spot !

**Comment by bjlittle** [comment]
Closing this PR, as it's gone stale after six months.

Replaced by #4490

**Pull request closed without merging**